vllm/engine/llm_engine.py

```diff
@@ -723,6 +725,44 @@ def _add_processed_request(
     def stop_remote_worker_execution_loop(self) -> None:
         self.model_executor.stop_remote_worker_execution_loop()
 
+    def process_model_params(
```

Comment:
I suggest adding this to `_build_logits_processors` instead of creating a new method. There is also no need to create an async version.

Reply:
Ok! I didn't notice `_build_logits_processors`. Indeed, it seems the right place for the "bad words". Moved logic here. Also removed the async version. I guess that really was an overkill)